### ISSUE TITLE
ASTC status

### ISSUE THREAD

**Issue by kelleyma49** [GammaUNC/FasTC#21]
I see a ASTCEncoder subdirectory, but I don't see command line options for targeting ASTC.  Am I correct in deducing that ASTC support is still work-in-progress? #

**Comment by Mokosha** [comment]
Yes, I've only currently written the decompressor to learn the format. ASTC is fundamentally similar to BC7 and DXT, although it has a lot more ability to select different bells and whistles which makes it a tad complicated to reason about. I suspect for now the (reference codec)[https://github.com/ARM-software/astc-encoder] is likely the best bet.

**Comment by sschuberth** [comment]
As a somewhat related question: If you already have the decoder implemented, could you make that part of this library, too? And more generally, not only include encoders but also decoders for the other formats, too?

**Comment by Mokosha** [comment]
That's the general idea. Most of the other formats have decoders, IIRC. The only one that doesn't is DXT5, but that's an easy one. ;)

**Comment by sschuberth** [comment]
Ah, that's good to hear. I was not sure about the decoder being included (and I did not check the source code, shame on me) because all the directories end in "*Encoder", hence I thought that really only the encoder is included.
